test/data/tweets.csv

```diff
@@ -0,0 +1,3 @@
+id_str,full_text,geo,created_at,user_id_str,user_screen_name,user_description,user_location,user_verified
+1201316965991799999,RT @OtherUser: There’s zero chance that 20 Republican senators have enough integrity to remove Trump from office.   But there are still 3 reasons to impeach him. A failure by Congress to respond to these abuses would effectively render the constitution meaningless.,,Mon Dec 02 02:58:42 +0000 2019,18199999,User123,Some profile content here.,NYC/Rockland County/Cape Cod ,True
```

Comment:
consider stripping all fields. but we can do this on the post-processing / tokenization. the additional storage costs should be negligible.